src/core/tests/frontend/paddle/test_models/gen_scripts/generate_sum.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import print_function
+import sys
+import paddle
+import paddle.fluid as fluid
+from save_model import saveModel
+
+paddle.enable_static()
+
+
+def sum(name: str, x, dtype=None):
+    out = fluid.layers.sum(x)
+    exe = paddle.fluid.Executor(paddle.fluid.core.CPUPlace())
```

Comment:
hi I think you have to convert x to a paddle.static.data object before feed into sum(), pls refer to other generation script.